### PR TITLE
Javascript javascript1 week2

### PR DESCRIPTION
**Pull request by dmytriigithub**

### CODE REVIEW

File: javascript/javascript1/week2/Flight booking fullname function.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+function getFullname(firstname, surname, useFormalName=false, male=true) {
+    if (useFormalName & male) {
+        return `Lord ${firstname} ${surname}`;
+    }else if (useFormalName & male == false) {
+        return `Ms ${firstname} ${surname}`;
+    }
+    return `${firstname} ${surname}`;
+}
+
+console.log(getFullname(fullname1, fullname2, true, false));
```
Comment (vulk29) [inline javascript/javascript1/week2/Flight booking fullname function.js]: This works smoothly! but what if I forget to type in my name? 
can you maybe add a check and a prompt to enter a full name?

File: javascript/javascript1/week2/Event application.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function getEventWeekDay(a) {
+    let days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+    let date = new Date(); 
+    date.setDate(date.getDate() + a);
+  
+    return days[date.getDay()];
+  }
+  
+console.log( getEventWeekDay(3)); // Sa
```
Comment (vulk29) [inline javascript/javascript1/week2/Event application.js:9]: nice! great job using the getDay function :D

File: javascript/javascript1/week2/Student manager.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+console.log(addStudentToClass('Tim'));
+console.log(addStudentToClass('Margrethe II'));
+
+
+
+function getNumberOfStudents() {
+  return `There are ${class07Students.length} studets in class 07`;
+}
+
+console.log(getNumberOfStudents());
```
Comment (vulk29) [inline javascript/javascript1/week2/Student manager.js:42]: great job! nested if else statements are always tricky to manage!

File: javascript/javascript1/week2/Weather wear.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    }else if (temperature >= 15 & temperature < 20) {
+        return `The temperature is ${temperature} degrees celsius. You should wear a sweater`;
+    }else if (temperature >= 20 & temperature < 30) {
+            return `The temperature is ${temperature} degrees celsius. You should wear shorts and a t-shirt`;
+    } else {
+        return `The temperature is ${temperature} degrees celsius. Better stay at home`;
+    } 
+}
+const clothesToWear = weatherWear(-18);
+console.log(clothesToWear); // Logs out: "shorts and a t-shirt"
```
Comment (vulk29) [inline javascript/javascript1/week2/Weather wear.js:19]: great job managing extreme cases! these are important when you are writing such if else clauses.

File: javascript/javascript1/week2/calculator Of Tax Percentage.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    }else if (a >= 100000 & a < 300000) {
+        return `Your income is ${a}kr and you have to pay 30% of tax percentage it is ${a * tax30}kr`;
+    }else if (a >= 300000) {
+        return `Your income is ${a}kr and you have to pay 50% of tax percentage it is ${a * tax50}kr`;
+    } else {
+        return `You should find a job`;
+    }   
+}
+
+console.log(calculatorOfTaxPercentage(134567890));
```
Comment (vulk29) [inline javascript/javascript1/week2/calculator Of Tax Percentage.js:20]: per month? :O

File: javascript/javascript1/week2/Candy helper.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    'sweet' : 0.5,
+    'chocolate' : 0.7,
+    'toffee' : 1.1,
+    'chewing-gum' : 0.03
+};
+
+function addCandy(candyType, weight) {
+    for (let key in candyTablePrices) {
+        if (candyType === key){
+            boughtCandyPrices.push(candyTablePrices[key] * weight);
```
Comment (vulk29) [inline javascript/javascript1/week2/Candy helper.js]: but what if i try to buy the wrong candy? there should be an else here :)

**Review by vulk29 — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week2/Candy helper.js (hunk 6 above)
Comment (dmytriigithub) [reply javascript/javascript1/week2/Candy helper.js]: Hello,
Thanks for the review. I'll make some corrections

**Pull request merged**